lib/rqrcode/qrcode/qr_code.rb

```diff
@@ -117,7 +129,8 @@ def initialize( string, *args )
 
       @data                 = string
 
-      mode                  = QRAlphanumeric.valid_data?( @data ) ? :mode_alpha_numk : :mode_8bit_byte
+      mode                  = QRMODE_NAME[(options[:mode] || '').to_sym]
```

Comment:
I wasn't aware you still support it. Everything below v2.0 has been deprecated horribly long.

We can use something else as the fallback value, like `'-'` or `'_'`?